lib/__tests__/__snapshots__/exportAssignment.test.ts.snap

```diff
@@ -0,0 +1,30 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`an export assignment as module member matches the snapshot 1`] = `
+declare module 'my-module' {
+    declare const test: string;
```

Comment:
Shouldn't the `declare` keyword be omitted here in this module scope? While investigating this I noticed that `ContextFlags.Module` has no effect on anything (if I didn't miss anything)!?